server/routes/juror-management/juror-record/juror-record.controller.js

```diff
@@ -443,6 +443,18 @@
           jurorOverview.data.commonDetails.poolNumber,
         );
 
+
+        const dates = attendance.juror_attendance_response_data.map(attendances => {
+          const [year, month, day] = attendances.attendance_date;
+
+          return new Date(year, month - 1, day);
+        });
+
+        const latestDate = new Date(Math.max(...dates));
+
+        const formattedDate = ('0' + latestDate.getDate()).slice(-2) +
+        '/' + ('0' + (latestDate.getMonth() + 1)).slice(-2) + '/' + latestDate.getFullYear();
```

Comment:
what is this trying to do? 😄

Reply:
The date is coming back in this format 2024-04-02T23:00:00.000Z, so it changes it into 02/04/2024 for easier processing later